scripts/build/rollup.docs.config.js

```diff
@@ -14,6 +14,6 @@ export default Object.assign(baseConfig, {
   format: "iife",
   plugins: [json(), resolve({ preferBuiltins: true }), commonjs(), globals()],
   useStrict: false,
-  moduleName: basename.replace(/.+-/, ""),
+  moduleName: basename.replace(/.+?-/, "").replace(/-/g, "_"),
```

Comment:
`.+?` can be written as `.*` (you don't have to change it, just an FYI)

Reply:
It's not --exactly-- the same. One is greedy and the other one is not. I don't think it matters in this case though :)

Reply:
I just want to use its full name as module name so as to avoid conflicts, though it's probably not going to be happened. 😅

- basename: `parser-json-stringify`
- with `.+-`: `stringify`
- with `.+?-`: `json_stringify`

Reply:
@vjeux huh, TIL!

Reply:
@ikatyang `*` means `0 or more`

Reply:
@vjeux `+?` means match as few of the preceding specifier as possible instead of the most possible, while still getting a match.

Reply:
Yeah, `?` has different meanings depending on the preceding thing.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#special-questionmark